Makefile

```diff
@@ -123,7 +123,7 @@ run-hub: generate manifests ## Run DR Orchestrator controller from your host.
 run-dr-cluster: generate manifests ## Run DR manager controller from your host.
 	go run ./main.go --config=examples/dr_cluster_config.yaml
 
-docker-build: test ## Build docker image with the manager.
+docker-build: ## Build docker image with the manager.
```

Comment:
Is this change related to the PR subject?

Reply:
No no... :) i just commented out test as I was merging and wanted to finish e2e, so this would need to be reverted if we are to merge this. More later...